### PR TITLE
History and category filter functionality

### PR DESCRIPTION
**Pull request by sushil-mod**
# Added History and Category filter functionality 
Now users can filter videos according to what category they like and also after clicking every video you can see the video in the history  

## What Files to check 
- history slice 
- categoryCard and categoryChips
- home page

### CODE REVIEW

File: src/components/chips/CategoryChips.css [hunk 1]
```diff
@@ -10,8 +10,13 @@
...
     border:1px solid #F5F5F5;
+    
 }
 .chips:hover{
     background-color: #F5F5F5;
     color: #4D4D4D;
+}
+.chip-active{
+    background-color: #F5F5F5;
+    color: #4D4D4D;
```
Comment (OmKakatkar) [inline src/components/chips/CategoryChips.css:21]: The colors seem to be repeated, you can use a variable here

File: src/features/history/historySlice.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import axios from "axios";
+
+const initialState = {
+    historyVideosLoader : false,
+    historyVideos : [],
+}
+
+export const addToHistoryVideos = createAsyncThunk("historyVideoList/addToHistoryVideos",async({video,userToken},{rejectWithValue})=>{
+    try {
+        console.log("history add",video,userToken);
```
Comment (OmKakatkar) [inline src/features/history/historySlice.js]: Remove console.log

**Review by OmKakatkar — APPROVED** [review]
Both History and Category are working fine

**Review by ganesh-kanchi — APPROVED** [review]
the functionalities are working fine. the code is very readable and the file structure is well maintained. this branch can be merged to dev.

**Pull request merged**